metabase/metabase
# Fix loading values for virtual fields

Fixes https://github.com/metabase/metabase/issues/30562
Fixes https://github.com/metabase/metabase/issues/31662
Reproduces https://github.com/metabase/metabase/issues/30562
Reproduces https://github.com/metabase/metabase/issues/31662

Previously the component tried to fetch field values for all fields, including ones that aren't backed by real DB fields. This PR fixes the issue and makes the component load data only for real fields.

How to verify:
- New -> Question -> Orders
- Add a custom column `[Total] + 1`
- Add a filter based on this column
- There should be no failed request for `values` as before
